build.xml

```diff
@@ -95,6 +95,13 @@
                 <exclude name="**/Makefile*"/>
             </fileset>
         </copy>
+        <exec executable="sed" failonerror="true" failifexecutionfails="true">
```

Comment:
I would move these code to `-pre-copy-cppTango` target, because this one intends to do dummy copying without side effects. Do not forget to change `${distrdir}` with `${builddir}`

Reply:
Done. I assume you meant to change `distrdir` to `workdir`, not to `builddir`, as `-pre-copy-cppTango` populates the former.